jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractMultiGridPresenter.java

```diff
@@ -150,6 +154,15 @@ protected void navigateToPerspective(final String perspectiveId,
                              parameterValue);
         placeManager.goTo(request);
     }
+    
+    public void openView(final String itemId) {
```

Comment:
This is probably too specific to be added in this class, ideally this should be in the target presenter classes.